specification/voiceservices/data-plane/Microsoft.VoiceServices/stable/2023-10-01/swagger.json

```diff
@@ -0,0 +1,1015 @@
+      ]
+    }
+  ],
+  "paths": {
+    "/account/{accountName}/numbers": {
+      "get": {
+        "operationId": "Account_GetNumbers",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "accountName",
+            "pattern": "^([A-Za-z0-9_-]+)$",
+            "minLength": 1,
+            "maxLength": 100,
+            "required": true,
+            "type": "string",
+            "description": "Name of the account. Can only contain letters, numbers, underscores and dashes. Can be up to 100 characters in length."
+          },
+          {
+            "format": "int32",
+            "in": "query",
+            "name": "maxpagesize",
+            "type": "integer",
+            "description": "Maximum number of records to include in a single response."
+          },
+          {
+            "format": "int32",
+            "in": "query",
+            "name": "skip",
+            "type": "integer",
+            "description": "An offset into the collection of the first record to be returned."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/NumberResponseDtoPaginatedListDto"
+            }
+          },
+          "default": {
+            "description": "An error response object.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/errorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get account numbers.": {
+            "$ref": "./examples/GetAccountNumbers.json"
+          }
+        },
+        "tags": [
+          "Number"
+        ],
+        "description": "Gets a list of the numbers assigned to the specified account. Limit the length of the list with the skip and maxpagesize parameters."
+      }
+    },
+    "/account/{accountName}/numbers/batch": {
+      "post": {
+        "operationId": "Account_UpdateNumbers",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "accountName",
+            "pattern": "^([A-Za-z0-9_-]+)$",
+            "minLength": 1,
+            "maxLength": 100,
+            "required": true,
+            "type": "string",
+            "description": "Name of the account. Can only contain letters, numbers, underscores and dashes. Can be up to 100 characters in length."
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/BatchNumbersPutDto"
+            }
+          }
```

Comment:
Should this body parameter be marked as required?

Reply:
Yes - went over all of the requests with bodies and fixed this.